app/_hub/kong-inc/acme/index.md

```diff
@@ -0,0 +1,213 @@
+      compatible:
+
+params:
+  name: acme
+  api_id: false
+  service_id: false
+  route_id: false
+  consumer_id: false
+  protocols: ['http', 'https', 'tcp', 'tls', 'grpc', 'grpcs']
+  dbless_compatible: yes
+  config:
+    - name: account_email
+      required: yes
+      default:
+      description: |
+        The account identifier, can be reused in different plugin instance.
+    - name: api_uri
+      required: false
+      default: "`https://acme-v02.api.letsencrypt.org`"
+      description: |
+        The ACMEv2 API endpoint to use, user might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/) during testing.
+    - name: cert_type
+      required: false
+      default: "`rsa`"
+      description: |
+        The certificate type to create, choice of `"rsa"` for RSA certificate or `"ecc"` for EC certificate.
+    - name: domains
+      required: false
+      default: "`[]`"
+      description: |
+        The list of domains to create certificate for. Wildcard domain like `*.example.com` is also supported. Regex pattern is not supported.
+    - name: renew_threshold_days
+      required: false
+      default: "`14`"
+      description: |
+        Days before expire to renew the certificate.
+    - name: storage
+      required: false
+      default: "`shm`"
+      description: |
+        The backend storage type to use, choice of `"kong"`, `"shm"`, `"redis"`, `"consul"` or `"vault"`. In dbless mode, `"kong"` storage is unavailable.
+    - name: storage_config
+      required: false
+      default:
+      description: |
+        Storage configs for each backend storage. See below for its default value.
+    - name: tos_accepted
+      required: false
+      default: "`false`"
+      description: |
+        If you are using Let's Encrypt, you must set this to true to agree the [Terms of Service](https://letsencrypt.org/repository/).
+  extra: |
+    `config.storage_config` is a table for all posisble storage types, by default it is:
+    ```lua
+        storage_config = {
+            kong = {},
+            shm = {
+                shm_name = kong
+            },
+            redis = {
+                host = '127.0.0.1',
+                port = 6379,
+                database = 0,
+                -- Redis authentication key
+                auth = nil,
+            },
+            consul = {
+                host = '127.0.0.1',
+                port = 8500,
+                -- kv prefix path
+                kv_path = "acme",
+                -- Consul ACL token
+                token = nil,
+                -- timeout in ms
+                timeout = 2000,
+            }
+            vault = {
+                host = '127.0.0.1',
+                port = 8200,
+                -- secrets kv prefix path
+                kv_path = "acme",
+                -- Vault token
+                token = nil,
+                -- timeout in ms
+                timeout = 2000,
+            },
+        }
```

Comment:
Please use JSON representation here as we do not use Lua code in the Admin API itself anywhere.

Reply:
updated with a json text

Reply:
updated with a json text